source/preferences/manage-your-desktop-notifications.rst

```diff
@@ -0,0 +1,86 @@
+Manage your desktop notifications
+=================================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. |dot-badge| image:: ../images/dot-badge.png
+  :alt: A dot on the badge means you have unread activity in at least one channel you're a member of.
+  :width: 50px
+
+.. |numbered-badge| image:: ../images/numbered-badge.png
+  :alt: A numbered badge means you have at least 1 unread message, @mention, or one of your keywords has triggered a notification.
+  :width: 50px
+
+Enable notifications
+--------------------
+
+From Mattermost v9.9 and desktop app v5.5, Mattermost prompts you to enable notifications in the desktop app the first time you connect to a Mattermost server.
+
+- When you select **Allow**, you won't be asked again. You'll start receiving notifications for all Mattermost activity with `badges <#badge-based-notifications>`__, `banner alerts <#banner-alerts>`__ and `sounds <#notification-sounds>`__. See the section below on `customizing your notifications <#customize-your-notifications>`__ based on how you prefer to be notified about Mattermost activity in the desktop app.
```

Comment:
we can also include here that when we click allow, we will recieve a browser banner which will ask for permission for notification

Reply:
![Screenshot 2024-10-01 at 10 27 56 AM](https://github.com/user-attachments/assets/a3334096-4995-4385-b20a-3b5dae1475fa)

Reply:
Yes i have one in [here](https://github.com/mattermost/docs/pull/7378#issuecomment-2355326433) as well